eng/scripts/CodeCheck.ps1

```diff
@@ -191,18 +191,25 @@ try {
         }
     }
 
-    Write-Host "Checking for changes to API baseline files"
+    $targetBranch = $env:PULLREQUEST_TARGETBRANCH
+    $sourceBranch = $env:PULLREQUEST_SOURCEBRANCH
 
     # Retrieve the set of changed files compared to main
-    $commitSha = git rev-parse HEAD
-    $changedFilesFromMain = git --no-pager diff origin/main...$commitSha --ignore-space-change --name-only --diff-filter=ar
+    Write-Host "Checking for changes to API baseline files $targetBranch...$sourceBranch"
+
+    $changedFilesFromMain = git --no-pager diff origin/$targetBranch...origin/$sourceBranch --ignore-space-change --name-only --diff-filter=ar
```

Comment:
Suggest pushing to the AzDO repo and trying out a PR there. Not positive what the name of the remote is there&hellip;

Reply:
Tested in the PR build: https://dev.azure.com/dnceng/internal/_build/results?buildId=1010689&view=logs&j=efda9e5f-cf98-536e-4181-9b6c13ac35b3&t=72de228c-ead8-57a1-ad6c-7744da2d78d6